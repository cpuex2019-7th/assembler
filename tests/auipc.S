    addi zero, zero, 0          ; nop
    addi zero, zero, 0          ; nop
    auipc a0, 0x7ffff           ; a0 will be 0x7ffff008
