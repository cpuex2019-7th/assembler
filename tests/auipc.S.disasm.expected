addi zero, zero, 0
addi zero, zero, 0
auipc a0, 524287
